src/eval/nnue/evaluate_nnue.cpp

```diff
@@ -23,7 +23,7 @@ AlignedPtr<FeatureTransformer> feature_transformer;
 AlignedPtr<Network> network;
 
 // Evaluation function file name
-const char* const kFileName = "nn.bin";
+const char* kFileName = "eval\\nn.bin";
```

Comment:
"k" is the prefix for a constant variable.  Could you please rename to "file_name" or something?